BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Extensibility/Implementation/Tracing/SelfDiagnostics/SelfDiagnosticsConfigRefresherTest.cs

```diff
@@ -74,6 +77,52 @@ public void SelfDiagnosticsConfigRefresher_CaptureAsConfigured()
             }
         }
 
+        [TestMethod]
+        public void SelfDiagnosticsConfigRefresher_ReadFromEnviornmentVar()
+        {
+
+            var key = "APPLICATIONINSIGHTS_LOG_DIAGNOSTICS";
+            var val = string.Empty;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                val = @"C:\home\LogFiles\SelfDiagnostics";
+            }
+            else
+            {
+                val = "/home/user/LogFiles/SelfDiagnostics";
+            }
```

Comment:
Minor: maybe simplify this to:

```suggestion
            var val = "/home/user/LogFiles/SelfDiagnostics";
            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
            {
                val = @"C:\home\LogFiles\SelfDiagnostics";
            }
```